lib/recipient.rb

```diff
@@ -0,0 +1,50 @@
+require 'httparty'
+require 'dotenv'
+
+class Recipient
+  attr_reader :slack_id, :name
+
+  class SlackApiError < Exception; end
+
+  def initialize(slack_id:, name:)
+    raise ArgumentError unless (slack_id && name)
+    @slack_id = slack_id
+    @name = name
+  end
+ 
+  def self.list
+    raise NotImplementedError, "template method"
+  end
+
+  def self.get(url, params)
+    response = HTTParty.get(url, query: params)
+    raise SlackApiError unless response['ok']
```

Comment:
It would be helpful to include the error in the message and also check the response code.

```suggestion
    raise SlackApiError.new(response["error"]) unless response['ok'] && response.code == 200
```